src/features/playlist/add-track/ui/AddTrackModal.module.css

```
@@ -57,15 +58,15 @@
 }
 
 .label {
-  color: #5d5d5d;
+  color: #8d8d8d;
   font-size: 16px;
   font-weight: var(--fw-regular);
   line-height: 1.4;
 }
 
 .youtubeLink {
   border-bottom: 1px solid currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)